everviz/pages/crossplot.py

```diff
@@ -0,0 +1,12 @@
+def page_layout(api):
+    everest_export = api.everest_csv
+    if everest_export is not None:
+        return {
+            "title": "Cross plots",
+            "content": [
+                {"Crossplot": {"data_path": everest_export,},},
+                {"CrossplotIndexed": {"data_path": everest_export,}},
+            ],
+        }
+    else:
+        return ""
```

Comment:
Does just returning an empty string work, shouldn´t it be an empty dict? And should we perhaps have a test for that? Other than that the PR looks good 👍 Are you planning any more changes?

Reply:
If you return an empty dict webviz will fail because it requires a title for its components, and I was thinking of not having any page there if the API cannot return a file path. 

A test should be there yes

Reply:
That makes sense 👍

Reply:
I am not planning any more changes